src/styles/parts/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-white-color;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card__apple {
```

Comment:
You don't use sass syntax. It should be
```
.card {
  &__apple{
  //
  }
} 
```
instead of css syntax:
```
.card {}
.card__apple {}
```
and so on in card.scss file. Other files look fine.